### DISCUSSION TITLE
getting NSInvalidArgumentException error in MacOs 14 sonoma while starting the server

### DISCUSSION THREAD

**Discussion by palaniappa** [milvus-io/milvus#28334] [Q&A and General discussion]
getting this error in MacOs 14 sonoma
Any help on this one?
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[__NSCFString stringByStandardizingPath]: unrecognized selector sent to instance 0x600000414580'
*** First throw call stack:
(
	0  CoreFoundation           0x00000001862ea800 __exceptionPreprocess + 176
	1  libobjc.A.dylib           0x0000000185de1eb4 objc_exception_throw + 60
	2  CoreFoundation           0x000000018639c3bc -[NSObject(NSObject) __retain_OA] + 0
	3  CoreFoundation           0x0000000186254a84 ___forwarding___ + 1572
	4  CoreFoundation           0x00000001862543a0 _CF_forwarding_prep_0 + 96
	5  Foundation             0x00000001873298a0 -[NSProcessInfo arguments] + 188
	6  CoreFoundation           0x0000000186365fd8 __getDefaultArguments_block_invoke + 96
	7  libdispatch.dylib          0x0000000185feb910 _dispatch_client_callout + 20
	8  libdispatch.dylib          0x0000000185fed14c _dispatch_once_callout + 32

**Comment by yhmo** [comment]
MacOS 14 is the latest version, we didn't get a chance to test the compatibility, yet.
Are you trying to run milvus on this OS by docker-compose?

**Reply by xiaofan-luan** [reply]
I think you might need Arm image to run it on macos?

**Comment by yhmo** [comment]
We just tested the Milvus v2.3.2 standalone on a MacOS 14 machine with Inter CPU. It works well, we didn't get any error.

**Comment by palaniappa** [comment]
Hi
No, I am not running in docker. I was trying to run locally. I am using Mac M1.

Thanks
Palani

**Reply by yhmo** [reply]
You compiled the source code?
